index.html

```diff
@@ -154,7 +154,7 @@ <h5>LET'S CONNECT</h5>
           <h3>Languages</h3>
           <a href="#" class="arrow"><i class="uil uil-angle-down"></i></a>
         </div>
-        <ul class="lang-img desktop">
+        <ul class="lang-img desktop flex">
```

Comment:
- [ ] Kindly make sure you stick to the Figma template 
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/192848068-2978caa2-3abd-4cec-9300-eb417cff1b3b.png)
Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/192848334-7bb8ad9c-2cfc-4ae7-b942-7d3655133105.png)

Reply:
Good job following the advice of the previous reviewer